Methods/SumNumbersDiagonals.java

```diff
@@ -0,0 +1,31 @@
+//Ունեք nxn չափի մատրիցա, n-ը որոշեք դուք։ Պետքա գրել ծրագիր, որը կհաշվի մատրիցայի անկյունագծերի
+// վրայի թվերի գումարը։
+
+package Methods;
+
+public class SumNumbersDiagonals {
+    public static void main(String[] args) {
+        int [][] matrix = {
+                {1, 7, 8, 6},
```

Comment:
int [][] matrix = {
                {1, 7, 8, 1,1},
                {2, 1, 1, 1,1},
                {4, 1, 1, 0,1},
                {1, 1, 8, 1,1},
                {1, 5, 8, 1,1}
        };
       The sum of the numbers on the diagonals is: 8
please check the logic again

Reply:
The Case is fixed